newm-server/src/main/kotlin/io/newm/server/Application.kt

```diff
@@ -23,7 +23,9 @@ import io.newm.server.staticcontent.createStaticContentRoutes
 import io.newm.server.statuspages.installStatusPages
 import io.newm.shared.daemon.initializeDaemons
 
-fun main(args: Array<String>) = io.ktor.server.cio.EngineMain.main(args)
+fun main(args: Array<String>) {
+    io.ktor.server.cio.EngineMain.main(arrayOf(*args, "-config=application.conf", "-config=${System.getenv("AWS_SECRET_ARN")}"))
```

Comment:
There is a little bit of awkwardness to avoid having to modify or extend the ktor classes.  Here the EnginMain and config loaders expect `-config=file arguments`